tests/v3/test_store/test_core.py

```diff
@@ -39,10 +39,7 @@ async def test_make_store_path(tmpdir: str) -> None:
 
 
 async def test_make_store_path_fsspec(monkeypatch) -> None:
-    import fsspec.implementations.memory
-
-    monkeypatch.setattr(fsspec.implementations.memory.MemoryFileSystem, "async_impl", True)
-    store_path = await make_store_path("memory://")
+    store_path = await make_store_path("http://foo.com/bar")
```

Comment:
using a real path here turned up a real bug in fsspec (reported in https://github.com/fsspec/filesystem_spec/issues/1722 and handled in `from_url` for now).

Reply:
Noting that this bug is the "expected" behavior in fsspec. Just not what we wanted.